src/extension/features/budget/date-of-money/index.js

```diff
@@ -0,0 +1,54 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteBudgetPage } from 'toolkit/extension/utils/ynab';
+
+export class DateOfMoney extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage() && document.querySelector('.toolkit-date-of-money') == null;
+  }
+
+  onRouteChanged() {
+    if (this.shouldInvoke()) {
+      this.invoke();
+    }
+  }
+
+  invoke() {
+    /*
+     * Get the parent div from ageOfMoneyContainer.
+     * This is used to:
+     * 		1. To get the Age Of Money (AOM)
+     * 		2. Display the Date of Money to the user
+     */
+    const budgetHeaderDaysContainer = document.querySelector('.budget-header-days');
+    /*
+     * Get the div containing the Age Of Money (AOM)
+     * If enabled, days of buffering has the same class, however AOM will always be the first element
+     */
+    const ageOfMoneyContainer = budgetHeaderDaysContainer.firstElementChild;
+    /*
+     * Get the Age Of Money
+     */
+    let ageOfMoney = 0;
+    const regExAOM = /\d+/;
+    const regExAOMResult = regExAOM.exec(ageOfMoneyContainer.innerText);
+    if (regExAOMResult != null) {
+      ageOfMoney = regExAOMResult[0];
+    } else {
+      return;
+    }
```

Comment:
Instead of this, you could use:

```
import { controllerLookup } ...

const budgetController = controllerLookup('budget');
const ageOfMoney = budgetController.get('budgetViewModel.monthlyBudgetCalculationForCurrentMonth.ageOfMoney');
```

From my testing, this will always grab age of money from the currently-viewed month so it should always be accurate.